.changeset/cold-chairs-fold.md

```diff
@@ -0,0 +1,7 @@
+---
+"firebase": major
```

Comment:
This should be `minor` with the new release schedule, right?

Reply:
Updated. Thanks!